newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -8,19 +8,8 @@
 # Do not add anything above this next line.
 common: &default_settings
 
-  # ============================== LICENSE KEY ===============================
-  # You must specify the license key associated with your New Relic
-  # account. For example, if your license key is 12345 use this:
-  # license_key: '12345'
-  # The key binds your Agent's data to your account in the New Relic service.
-  license_key: '<%= license_key %>'
-
-  # Agent Enabled
-  # Use this setting to disable the agent instead of removing it from the startup command.
-  # Default is true.
-  agent_enabled: true
-
-  # Set the name of your application as you'd like it show up in New Relic.
+  # ============================== APP NAME ===============================
```

Comment:
My guess is that license_key is the first config item because it is the one required thing that is not provided by default to get up and running.  An app_name is provided by default, so I wonder if license_key being the first config option serves the customer better.  Is there a justification that I'm not aware of that makes app_name a better choice?
@jasonjkeller @meiao and/or @twcrone can someone else weigh in on this one as well?

Reply:
My rationale for putting them in this order is that in the UI and in the docs, we tend to talk about app name first, before talking about license key. @tbradellis

Reply:
I chatted with the team about this and they agree with my rationale.  We'd like the license key as it is currently (first under common) because it is the one thing not already provided in the default config that needs to be provided for the agent to start.